build-scripts/install_tensorflow.sh

```diff
@@ -0,0 +1,8 @@
+mkdir -p /root/src/tflite
```

Comment:
It requires the root privilege to access the `/root` directory, please clone the code under `<wamr_root>/core/deps` instead.